README.md

```diff
@@ -0,0 +1,75 @@
+<p align="center">
+ <img src="https://user-images.githubusercontent.com/62607343/202245103-044d5c0b-7dec-416b-a178-66b4875dc399.png" width="200" alt="SAIL Logo" /></a>
+</p>
+
+
+
+# NOTIFICATION-SERVICE
+
+This is a notification microservice that lets you create email templates, queue your email on AWS SQS, and send the email to users of your applications through AWS SES.
+
+
+## Packages
+<li> <a href="https://github.com/hicsail/notification-service/tree/readMe/packages/server/src/email"> Email Notification Service </a>
+<li> <a href="https://github.com/hicsail/notification-service/tree/readMe/packages/server/src/templates"> Email Template Service </a>
+<li> <a href="https://github.com/hicsail/notification-service/tree/readMe/packages/server/src/health">  Health Service </a>
+
+## 📖 Prerequisite
+
+In order to run/develop this Microservice, your project needs the following dependecies:
+
+<pre>
+"node": "^14.15.0 || >=16.10.0",
+"@ssut/nestjs-sqs": "^1.3.0",
+"aws-sdk": "^2.1227.0",
+"dotenv": "^16.0.3",
+"sqs-producer": "^2.1.0",
+"uuid": "^9.0.0"
+</pre>
+
+Please also create .env file in your project and configure the following environmental variables:
+  
+<pre>
+AWS_REGION="us-east-1"
+AWS_ACCESS_KEY_ID=""
+AWS_SECRET_ACCESS_KEY=""
+SQS_QUEUE_URL =""
+</pre>
+  
+## Installation
```

Comment:
NIT: Should instruct the user to clone first rather then make the environment file. Otherwise they are then moving files after the clone

Reply:
Thanks. I moved installation to above the prerequisite